FlaskApp/templates/base.html

```diff
@@ -21,7 +21,10 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ"
         crossorigin="anonymous">
+        <link href="//cdn.quilljs.com/1.3.6/quill.core.css" rel="stylesheet">
```

Comment:
Remove these from your base

This will cause every page to load Quill's CSS and JS files, increasing page load time